library_generation/owlbot/src/fix-poms.py

```diff
@@ -477,52 +493,57 @@ def main(versions_file, monorepo):
             update_bom_pom(f"{artifact_id}-bom/pom.xml", modules)
     elif artifact_id+"-bom" not in excluded_poms_list:
         print("creating missing bom pom.xml")
+        monorepo_version = __get_monorepo_version(versions_file) \
```

Comment:
I may missed this in earlier reviews, can you please remind me why we need to get `monorepo_version` now?

Reply:
I want to generate the pom.xml in `*-bom` without using a repo-level post processing script (`set_parent_pom.sh` in this case) and run `mvn fmt:format` after the pom generation.

This requires to know the current version of `gapic-libraries-bom` and `google-cloud-jar-parent`, which is the same as the monorepo version.